Sources/SpeziSpeechRecognizer/SpeechRecognizer.swift

```diff
@@ -213,3 +213,9 @@ public final class SpeechRecognizer: NSObject, Module, DefaultInitializable, Env
         self.isAvailable = available
     }
 }
+
+
+/// Necessary `Sendable` conformance for Swift 6 language mode.
```

Comment:
Actually we are opting out here. Just curious why? 
(I don't have my Mac now 😅)

Reply:
I think the answer is just blow: 
> Note: Wrapping the `SFSpeechRecognitionResult` in a custom `Sendable` struct is not desired as it would break API.

right, @philippzagar?

Reply:
I think the answer is just below:
> Note: Wrapping the `SFSpeechRecognitionResult` in a custom `Sendable` struct is not desired as it would break API.

right, @philippzagar?

Reply:
Not quite, the issue is that we don't control 'SFSpeechRecognitionResult' (we don't even have access to source), meaning the compiler cannot verify 'Sendable' conformance at all.
That's why one needs to specify an 'unchecked' 'Sendable' conformance.

The alternative would be to mark the entire 'Speech' framework as '@preconcurrency' to disable automatic isolation checks for types of the framework.